docs/source/extend_kedro/plugins.md

```diff
@@ -29,8 +29,15 @@ def to_json(metadata):
     pipeline = pipelines["__default__"]
     print(pipeline.to_json())
 ```
+Starting from 0.18.14, Kedro switch to replace `setup.py` with `pyproject.toml`. The plugin need to provide the entry points in either file.
```

Comment:
```suggestion
From version 0.18.14, Kedro replaced `setup.py` with `pyproject.toml`. The plugin needs to provide entry points in either file.
```

Reply:
@stichbury We have moved to `pyproject.toml` if you created starter with the new version, but there are projects created with older version of Kedro and they only upgrade `kedro` but not the project template. Maybe its's too confusing and I should just keep the new one?